src/rdkafka.c

```diff
@@ -270,7 +279,463 @@ void rd_kafka_log0 (const rd_kafka_conf_t *conf,
         rd_kafka_log_buf(conf, rk, level, fac, buf);
 }
 
+static int check_oauthbearer_extension_key(const char *key,
+                char *errstr, size_t errstr_size) {
+        if (!strcmp(key, "auth")) {
+		rd_snprintf(errstr, errstr_size,
+		        "Cannot explicitly set the reserved `auth` SASL/OAUTHBEARER extension key");
```

Comment:
keep lines under 80 columns, break it like so:
```
		rd_snprintf(errstr, errstr_size,
		        "Cannot explicitly set the reserved `auth` "
                         "SASL/OAUTHBEARER extension key");
```

Reply:
👍